src/scripts/main.js

```diff
@@ -1,3 +1,246 @@
+
+labelName.textContent = 'Name: ';
+labelPosition.textContent = 'Position: ';
+labelOffice.textContent = 'Office: ';
+labelAge.textContent = 'Age: ';
+labelSalary.textContent = 'Salary: ';
+formButton.textContent = 'Save to table';
+
+const inputName = document.createElement('input');
+const inputPosition = document.createElement('input');
+const selectOffice = document.createElement('select');
+const inputAge = document.createElement('input');
+const inputSalary = document.createElement('input');
+
+inputName.required = true;
+inputName.name = 'name';
+inputName.type = 'text';
+inputName.dataset.qa = 'name';
+inputName.id = 'name';
+labelName.setAttribute('for', 'name');
+
+inputPosition.required = true;
+inputPosition.name = 'position';
+inputPosition.type = 'text';
+inputPosition.dataset.qa = 'position';
+inputPosition.id = 'position';
+labelPosition.setAttribute('for', 'position');
+
+selectOffice.required = true;
+selectOffice.name = 'office';
+selectOffice.dataset.qa = 'office';
+selectOffice.id = 'office';
+
+inputAge.required = true;
+inputAge.name = 'age';
+inputAge.type = 'number';
+inputAge.dataset.qa = 'age';
+inputAge.id = 'age';
+labelAge.setAttribute('for', 'age');
+
+inputSalary.required = true;
+inputSalary.name = 'salary';
+inputSalary.type = 'number';
+inputSalary.dataset.qa = 'salary';
+inputSalary.id = 'salary';
+labelSalary.setAttribute('for', 'salary');
+
+const offices = [
+  'Tokyo',
+  'Singapore',
+  'London',
+  'New-York',
+  'Edinburg',
+  'San Francisco',
+];
+
+offices.forEach((office) => {
+  const option = document.createElement('option');
+
+  option.value = office;
+  option.textContent = office;
+  selectOffice.appendChild(option);
+});
+
+newForm.appendChild(labelName);
+newForm.appendChild(inputName);
+newForm.appendChild(labelPosition);
+newForm.appendChild(inputPosition);
+newForm.appendChild(labelOffice);
+newForm.appendChild(selectOffice);
+newForm.appendChild(labelAge);
+newForm.appendChild(inputAge);
+newForm.appendChild(labelSalary);
+newForm.appendChild(inputSalary);
+newForm.appendChild(formButton);
+
+sortedRowsToArr.forEach((row) => {
+  row.addEventListener('click', () => {
+    sortedRowsToArr.forEach((r) => r.classList.remove('active'));
+    row.classList.add('active');
+  });
+});
+
+headerTitle.forEach((head) => {
+  head.addEventListener('click', (e) => {
+    const indexRow = e.target.cellIndex;
+
+    if (lastSortedColumnIndex === indexRow) {
+      isAsc = !isAsc;
+    } else {
+      isAsc = true;
+    }
+
+    lastSortedColumnIndex = indexRow;
+
+    sortedSort(sortedRowsToArr, indexRow);
+  });
+});
+
+formButton.addEventListener('click', (e) => {
+  e.preventDefault();
+
+  const newRow = document.createElement('tr');
+  const employeeName = inputName.value;
+  const position = inputPosition.value;
+  const office = selectOffice.value;
+  const age = inputAge.value;
+  const salaryValue = parseFloat(inputSalary.value);
+  const salary = '$' + salaryValue.toFixed(3);
```

Comment:
try to format it in the same way as current salaries:
![image](https://github.com/user-attachments/assets/ae1fc423-6ae1-4d6e-87df-b3868e0ebfa6)

you can try to use `toLocalString` method